24.mender-snapshot/docs.md

```diff
@@ -0,0 +1,8 @@
+---
+title: mender-connect
```

Comment:
shouldn't the title be `mender-snapshot`?

Reply:
Thank you

Reply:
https://github.com/mendersoftware/mender-docs-changelog/pull/59